src/main/services/bs-launcher/steam-launcher.service.ts

```diff
@@ -77,8 +77,11 @@ export class SteamLauncherService extends AbstractLauncherService implements Sto
                 throw CustomError.fromError(new Error(`Path not exist : ${exePath}`), BSLaunchError.BS_NOT_FOUND);
             }
 
+            const staticConfigService = StaticConfigurationService.getInstance();
+            const launchVREnabled = await staticConfigService.get("disable-launch-vr");
```

Comment:
Remove the `staticConfigService` variable and use the `this.staticConfig` member variable instead.